osu.Game/Screens/SelectV2/BeatmapPanel.cs

```diff
@@ -104,7 +425,10 @@ protected override bool OnClick(ClickEvent e)
 
         public double DrawYPosition { get; set; }
 
-        public void Activated() => activationFlash.FadeOutFromOne(500, Easing.OutQuint);
+        public void Activated()
+        {
+            // activationFlash.FadeOutFromOne(500, Easing.OutQuint);
```

Comment:
just leave the flash for now if we have nothing better? right now it does nothing.